config.yml

```diff
@@ -5,7 +5,7 @@ indices:
   days: 14
   # OPTIONAL: Cron timestamp (sec, min, hour, day of month, month, day of week) in UTC.
   # If specified, clears the old indices at that interval.
-  clearAt: "0 0 9 * * *" # 2 am PT is 9 am UTC
+  clearAt: "0 5 9 * * *" # 2:05 am PT is 9:05 am UTC
```

Comment:
Running it at night (or low traffic times) minimizes the chance of shard relocations causing ingest issues due to congestion. But we could move it toward the end of working hours if there is value.